exercises/es/exc_02_13.py

```diff
@@ -15,8 +15,13 @@
 )
 
 # Crea los patrones
-pattern1 = [{"LOWER": "Bandai"}, {"IS_TITLE": True, "POS": "PROPN"}]
-pattern2 = [{"LOWER": "Pac-Man"}, {"POS": "NOUN"}]
+pattern1 = [{"LIKE_NUM": False}, {"POS": "INTJ"}]
+pattern2 = [
+   {"LOWER": "pac"},
+   {"TEXT": "-"},
+   {"LOWER": "man"},
+   {"POS": "PROPN", "OP": "*"},
+]
```

Comment:
I like that solution 👍  Since `pattern2` is now quite long more abstract than before and auto-formatted over multiple lines, maybe we can simplify it a bit, like this? This should also make it fit on one 80-character line:

```python
pattern2 = [{"LOWER": "pac"}, {"TEXT": "-"}, {"LOWER": "man"}, {"LOWER": "Live"}]
```